### PR TITLE
fix bower issue

### PR DESCRIPTION
**Pull request by h8rry**

### CODE REVIEW

File: js/splitter.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
+/*
```
Comment (h8rry) [inline js/splitter.js:1]: Are you using this file? or are you using the files in bower_components?

File: index.html [hunk 2]
```diff
@@ -39,20 +39,22 @@
 			</article>
```
Comment (h8rry) [inline index.html:39]: Your indentations are still off

File: css/custom.css [hunk 3]
```diff
@@ -2,40 +2,50 @@ html, body, section {
     height: 800px;
```
Comment (h8rry) [inline css/custom.css:2]: Your indentations are still off

File: js/app.js [hunk 4]
```diff
@@ -1,57 +1,103 @@
...
-  //initialize properties of each grid cell (USE CLASS CONSTRUCTOR)
-  $('.per_cell').each(function(index, element){
-    this.pos = [index%12, $(element).parent().index()]; // [x,y] coords
-    this.owner = ['player 1', 'player 2', 'neutral']; //territory owner
-    this.garrison = 0;//initial garrison number;
-  });
+    //Initialize movement range with move_range function
+    $('.province').each(function(){
+      var mvrange = move_range(this.pos); 
+      this.mvrange = mvrange;
```
Comment (h8rry) [inline js/app.js:60]: Just one liner would be fine

`this.mvrange = move_range(this.pos);`

File: js/splitter.js (hunk 1 above)
Comment (h8rry) [inline js/splitter.js:1]: Using `bower install --save jquery.splitter`

Comment (h8rry) [inline js/splitter.js:1]: Let's remove this file

File: index.html (hunk 2 above)
Comment (jonycfu) [inline index.html:39]: fixed

File: css/custom.css (hunk 3 above)
Comment (jonycfu) [inline css/custom.css:2]: fixed

File: js/app.js (hunk 4 above)
Comment (jonycfu) [inline js/app.js:60]: fixed

**Pull request merged**